### DISCUSSION TITLE
How to reduce end to end latency

### DISCUSSION THREAD

**Discussion by felixyangtw** [bluenviron/mediamtx#903] [Questions]
I use a android phone to rtmp streaming to rtsp simple server and use  ffplay -fflags nobuffer -rtsp_transport tcp to play the stream, and measure the latency by watching the clock. I found that rtsp simple server is longer the other rtsp server (red5 pro server) abount 400 ms. Is there any parameter  or method to recude end to end latency?

**Comment by Curid** [comment]
The delay is caused by whatever android app you're using to create the stream.

**Reply by felixyangtw** [reply]
I use the same app to streaming to another (red5 pro server), the latency is lower than rtsp simple server about 400ms. Is there any parameter to reduce the latency (buffer size...) in rtsp simple server?

**Reply by aler9** [reply]
There is no delay inside the server.

[a comment by an automated account is not shown]
